docs/project/changelog.md

```diff
@@ -56,6 +56,8 @@ substitutions:
 - {{ API }} Added `PyProxy.getBuffer` API to allow direct access to Python
   buffers as Javascript TypedArrays.
   [1215](https://github.com/iodide-project/pyodide/pull/1215)
+- {{ API }} Added support for multiindexes with `PyProxy.get`, `PyProxy.set`, etc.
```

Comment:
multiindexing can have a bit different meaning in scientific python https://pandas.pydata.org/pandas-docs/stable/user_guide/advanced.html 

```suggestion
- {{ API }} Added support indexing with tuples using `PyProxy.get`, `PyProxy.set`, etc.
  For instance, `X.get(`
```